README.md

```diff
@@ -4,6 +4,9 @@
   <p align="center">Build, develop, and deploy Next.js apps for Cloudflare Pages.</p>
 </p>
 
+> [!NOTE]
+> You may want to consider using [`@opennextjs/cloudflare`](https://opennext.js.org/cloudflare), which allows you to build and deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers/static-assets/), use [Node.js APIs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/) that Cloudflare Workers supports, and supports additional Next.js features.
```

Comment:
```suggestion
> You may want to consider using [`@opennextjs/cloudflare`](https://opennext.js.org/cloudflare), which allows you to build and deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers/static-assets/), use [Node.js APIs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/) that Cloudflare Workers supports, and enables additional Next.js features like Incremental Static Regeneration.
```

I think it's worth specifically calling this one out as a reason, since it's the biggest drawback of next-on-pages

Reply:
Makes sense. Can add a few more things to this list soon :)